### ISSUE TITLE
Support for Bedrock accounts

### ISSUE THREAD

**Issue by benrobson** [dommilosz/minecraft-auth#9]
Hey,
I have just started working with this library, I have both a Java and Bedrock account and it doesn't list both of my accounts under `account.profile` is there something I am missing or can this be added?

**Comment by dommilosz** [comment]
So this authenticator is made for java edition. I follow this guide on how to authenticate: https://wiki.vg/Microsoft_Authentication_Scheme.
I think bedrock edition authentication is not possible that way because bedrock has no launcher and authenticates within itself. I probably would be able to get username of bedrock account but cannot forward it to game in order to play later.

**Comment by benrobson** [comment]
Yeah cool, I was just looking to see if there was some way to do this inside of the package to authenticate them?
I've basically got a website that requires the user to sign into their Microsoft account and allow the app to verify game ownership.

**Comment by dommilosz** [comment]
It is possible but idk if it fits this package. It was made to help launcher developers launch Minecraft with selected account. I will do some research on it and if it wouldn't increase size of this package a lot I will add it as side feature
